iTunesSearch/ItunesSearch.m

```diff
@@ -456,6 +456,29 @@ - (void)getAppsByDeveloper:(NSString *)developerId limitOrNil:(NSNumber *)limit
     }
 }
 
+- (void)getAppsWithIds:(NSArray *)appIds successHandler:(ItunesSearchReturnBlockWithArray)successHandler failureHandler:(ItunesSearchReturnBlockWithError)failureHandler {
+    
+    // Build a string of App Ids
+    NSString *appIdsString = [appIds componentsJoinedByString:@","];
+    
+    if (appIdsString && [appIdsString length] > 0) {
+        // Set up the request paramters
+        NSMutableDictionary *params = [[NSMutableDictionary alloc] init];
```

Comment:
Since the params never change it would be better to simply create one like this:

```
NSDictionary * params = @{
    @"id": [self forceString:appIdsString],
    @"media": @"software"
}
```